doc/source/API/Stackup3D.rst

```diff
@@ -0,0 +1,21 @@
+Stackup 3D Components
+=====================
+This section lists classes for creating and editing a stackup and objects the 3D tools.
+This consists of a set of one or more parametrized layer objects, where to place line, patch, polygon and vias.
```

Comment:
```suggestion
This section lists classes for creating and editing a stackup and objects in the 3D tools.
This consists of a set of one or more parametrized layer objects, where to place line, patch, polygon, and vias.
```

Reply:
I don't know what "This" refers to in the second sentence: "This consists of ...." Would you please rewrite this sentence to help clarify?